update_all/after/execution_output.txt

```diff
@@ -13,26 +13,11 @@ Finding Podfile changes
 Resolving dependencies of `Podfile`
 
 Comparing resolved specification to the sandbox manifest
-  M Reachability
+  - Reachability
 
 Downloading dependencies
 
--> Installing Reachability 3.1.0 (was 2.0.5)
- > Git download
- > Git download
-     $ GIT_BIN clone https://github.com/tonymillion/Reachability.git PROJECT/Pods/Reachability --template= --single-branch --depth 1 --branch v3.1.0
-     Cloning into 'PROJECT/Pods/Reachability'...
-     Note: checking out 'f7176f4798d068d233dca5223ae4bd9c8059e830'.
-
-     You are in 'detached HEAD' state. You can look around, make experimental
-     changes and commit them, and you can discard any commits you make in this
-     state without impacting any branches by performing another checkout.
-
-     If you want to create a new branch to retain commits you create, you may
-     do so (now or later) by using -b with the checkout command again. Example:
-
-       git checkout -b new_branch_name
-
+-> Using Reachability (3.1.0)
```

Comment:
I dont think this is right either, we're no longer testing that we're updating anything